test/test.py

```diff
@@ -152,52 +77,46 @@ def test_get_rate_coefficients(mechname, usr_np):
                            if sol.species_name(sp) not in
                            r.third_body.efficiencies])
             k_ct[i] *= eff
-
         print(k_ct)
         print()
         print(k_pm)
         print()
         print(np.abs((k_ct-k_pm)/k_ct))
         print(np.linalg.norm((k_ct-k_pm)/k_ct, np.inf))
         assert np.linalg.norm((k_ct-k_pm)/k_ct, np.inf) < 1e-13
-    return
 
 
 @pytest.mark.parametrize("mechname", [
     "uiuc", "sandiego", "uconn32", "gri30", "hong"
 ])
-@pytest.mark.parametrize("usr_np", numpy_list)
-def test_get_pressure(mechname, usr_np):
+@pytest.mark.parametrize("user_np", numpy_list)
+def test_get_pressure(mechname, user_np, request):
```

Comment:
Likewise, add a description of `request` to the docstring that follows.

Reply:
See the other comment on this.